Tests/RestKitTests/JSONValueTests.swift

```diff
@@ -57,12 +59,23 @@ class JSONTests: XCTestCase {
     ]
 
     func testEquality() {
+        // date values
+        let dateFormatter = DateFormatter()
+        dateFormatter.calendar = Calendar(identifier: .iso8601)
+        dateFormatter.locale = Locale(identifier: "en_US_POSIX")
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSXXXXX"
+        let stringDate = "2018-05-17T18:45:32.189Z"
+        let stringDate2 = "2017-04-16T15:45:32.189Z"
+        let date = dateFormatter.date(from: stringDate)
+        let date2 = dateFormatter.date(from: stringDate2)
+
         // equal values
         XCTAssertEqual(JSON.null, JSON.null)
         XCTAssertEqual(JSON.boolean(true), JSON.boolean(true))
         XCTAssertEqual(JSON.string("hello"), JSON.string("hello"))
         XCTAssertEqual(JSON.int(1), JSON.int(1))
         XCTAssertEqual(JSON.double(0.5), JSON.double(0.5))
+        XCTAssertEqual(JSON.date(date!), JSON.date(date!))
```

Comment:
Force unwraps in tests are bad because they blow up the whole testsuite. Instead, do this at the start of the function:

```swift
let date = dateFormatter.date(from: stringDate)
guard let date = date else {
    XCTFail("Failed to construct date")
    return
}
```

Reply:
Will do.